src/types.rs

```diff
@@ -3058,6 +3138,31 @@ impl TRS {
             .join("\n")
     }
     /// A human-readable serialization of the `TRS`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, parse_trs};
+    /// let mut sig = Signature::default();
+    ///
+    /// let trs = parse_trs(&mut sig, 
+    /// "A(x_ y_ z_) = A(x_ SUCC(ZERO) SUCC(SUCC(ZERO)));
+    /// CONS(B CONS(C CONS(D NIL))) = CONS(C CONS(D NIL));
+    /// B C D E = B C | D E;") 
+    ///     .expect("parse of A(x_ y_ z_) = A(x_ SUCC(ZERO) SUCC(SUCC(ZERO)));
+    ///     CONS(B CONS(C CONS(D NIL))) = CONS(C CONS(D NIL));
+    ///     B C D E = B C | D E;"); 
+    ///
+    /// assert_eq!(trs.display(&sig), 
```

Comment:
We don't need the `TRS::display` statement here, as it's already in the `TRS::display` test. If you agree, please remove.